webnode/package.json

```diff
@@ -25,12 +25,11 @@
     "iota.lib.js": "^0.4.7",
     "jest": "^23.1.0",
     "js-sha3": "^0.7.0",
+    "lodash": "^4.17.5",
     "node-forge": "^0.7.5",
     "object-assign": "4.1.1",
     "postcss-flexbugs-fixes": "3.2.0",
```

Comment:
Couple possible issues with this file.  I think we want to keep it as `"datamap-generator": "git+https://github.com/oysterprotocol/datamap-generator.git",` unless we decided to change it.  I also think someone recently pulled out lodash, but perhaps we decided to add it back.  I also think we'd want to keep preact?  

@jfarago

Reply:
@rfornea we dropped it from datamap-gen but not webnode. its small enough that its not a big deal now that we got a rid of other dependencies.

Reply:
Ok, so now its question if we prefer 
"datamap-generator": "git+https://github.com/oysterprotocol/datamap-generator.git",	
or 
"datamap-generator": "^1.0.6",

I tested it with ^1.0.6

Reply:
I would leave it as `"datamap-generator": "git+https://github.com/oysterprotocol/datamap-generator.git",` as that will always pull the latest master from datamap-generator and we don't have to rely on people publishing the npm package.

Reply:
Unless there is some performance-related reason not to do it this way.

Reply:
it make sense.. I will keep link to oyster datamap repo

Reply:
@jfarago @EdmundMai @AaronVasquez Do you know of any performance-related reason not to just pin this to whatever is in master (and not even bother with npm)?

Reply:
Once we get auto npm publishing working, we will switch the datamap-generator reference to semver

Reply:
not performance related, but each local copy can go out of sync. We can get into a state where each dev has a diff version. Not a big deal right now since we just yarn upgrade (and should always be doing so)

Reply:
@jfarago @rfornea @AaronVasquez ... when I am trying download datamap-generator exactly from git it doesnt work.. We will need probably use token auth for it. Its falling because it cant access.

So the question is.. if we really want to use datamap-generator exactly from repository, who can generate token for it? Alex Firmani?

After this change is pull request finished.

Reply:
I have not had this issue.  I wonder why you are having this problem and other people aren't?

Reply:
When I am using exactly our repository, its failing... details here: https://travis-ci.org/oysterprotocol/webnode/builds/398196742